view/frontend/web/js/view/payment/method-renderer/omise-offsite-installment-method.js

```diff
@@ -165,6 +234,32 @@ define(
                 );
             },
 
+            /**
+             * Get installment term ko.observable by name
+             *
+             * @return {string|null}
+             */
+            getObservableTrem: function (name) {
```

Comment:
should this be getObservableTerm?

Reply:
nice catch !!
Fixed -> https://github.com/omise/omise-magento/pull/337/commits/72d9606215fd4c67893798f6857f0487081c6435 🙏